docs/connect.html

```diff
@@ -283,7 +283,7 @@ <h4><a id="connect_rest" href="#connect_rest">REST API</a></h4>
         <li><code>ssl.client.auth</code></li>
     </ul>
 
-    <p>The REST API is used not only by users to monitor / manage Kafka Connect. It is also used for the Kafka Connect cross-cluster communication. Requests received on the follower nodes REST API will be forwarded to the leader node REST API.
+    <p>The REST API is used not only by users to monitor / manage Kafka Connect. In distributed mode, it is also used for the Kafka Connect cross-cluster communication. Some requests received on the follower nodes REST API will be forwarded to the leader node REST API.
```

Comment:
Do we want to document which requests are forwarded and which ones aren't or do you feel like that's unnecessary in these docs?

Reply:
Definitely not necessary; users should not have to worry about this for the most part.